chatterbot/storage/django_storage.py

```diff
@@ -181,8 +181,8 @@ def add_to_conversation(self, conversation_id, statement, response):
         Statement = self.get_model('statement')
         Response = self.get_model('response')
 
-        first_statement = Statement.objects.get(text=statement.text)
-        first_response = Statement.objects.get(text=response.text)
+        first_statement, created = Statement.objects.get_or_create(text=statement.text)
```

Comment:
But the changes are looks good, but why you are storing into another variables? Could please elaborate?

Reply:
Do you mean about `created` ? Its because `get_or_create` returns a tuple of (object, created). No apparent use of `created` but still need keep it there to separate out the variables.